### PR TITLE
update images-list for v3.19.0 calico images

### PR DESCRIPTION
**Pull request by kinarashah**
#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags in the images-list file
- [x] Change does not remove / overwrite exiting Images or Tags in Rancher DockerHub
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexicographically)
- [x] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
- [x] Changes to scripting or CI config have been tested to the best of your ability

#### Linked Issues ####

https://github.com/rancher/rancher/issues/31567

### CODE REVIEW

File: images-list [hunk 1]
```diff
@@ -220,24 +220,29 @@ quay.io/calico/cni rancher/mirrored-calico-cni v3.13.4
 quay.io/calico/cni rancher/mirrored-calico-cni v3.16.5
 quay.io/calico/cni rancher/mirrored-calico-cni v3.17.2
 quay.io/calico/cni rancher/mirrored-calico-cni v3.18.1
+quay.io/calico/cni rancher/mirrored-calico-cni v3.19.0
 quay.io/calico/ctl rancher/calico-ctl v3.16.5
 quay.io/calico/ctl rancher/calico-ctl v3.16.6
 quay.io/calico/ctl rancher/calico-ctl v3.17.1
 quay.io/calico/ctl rancher/calico-ctl v3.17.2
 quay.io/calico/ctl rancher/calico-ctl v3.18.1
+quay.io/calico/ctl rancher/calico-ctl v3.19.0
```
Comment (StrongMonkey) [inline images-list:229]: Do we need this since it is already switched to mirrored-?

Comment (kinarashah) [reply images-list:229]: I don't think so, removing

File: images-list [hunk 2]
```diff
@@ -247,15 +251,18 @@ quay.io/calico/node rancher/mirrored-calico-node v3.13.4
 quay.io/calico/node rancher/mirrored-calico-node v3.16.5
 quay.io/calico/node rancher/mirrored-calico-node v3.17.2
 quay.io/calico/node rancher/mirrored-calico-node v3.18.1
+quay.io/calico/node rancher/mirrored-calico-node v3.19.0
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.16.5
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.16.6
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.17.1
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.17.2
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.18.1
+quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.19.0
```
Comment (StrongMonkey) [inline images-list:260]: Also this?

File: images-list [hunk 3]
```diff
@@ -229,15 +230,18 @@ quay.io/calico/ctl rancher/mirrored-calico-ctl v3.13.4
 quay.io/calico/ctl rancher/mirrored-calico-ctl v3.16.5
 quay.io/calico/ctl rancher/mirrored-calico-ctl v3.17.2
 quay.io/calico/ctl rancher/mirrored-calico-ctl v3.18.1
+quay.io/calico/ctl rancher/mirrored-calico-ctl v3.19.0
 quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.16.5
 quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.16.6
 quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.17.1
 quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.17.2
 quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.18.1
+quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.19.0
```
Comment (StrongMonkey) [inline images-list:239]: same?

**Review by StrongMonkey — APPROVED** [review]
Not sure if quay.io/calico/typha is used, LGTM

**Review by rmweir — APPROVED** [review]
LGTM

**Review by cbron — APPROVED** [review]
lgtm, we can merge in morning

**Pull request merged**